Snake.php

```diff
@@ -327,4 +339,14 @@ public function printGame()
 
     echo "\e[?25l";
   }
+
+  private function startMusic($musicFile)
+  {
+    if (extension_loaded('modplayer')) modplayer_play($this->resourcesFolder . '/music/' . $musicFile);
```

Comment:
Can we get these changed to:

```php
if (extension_loaded('modplayer')) {
    modplayer_play(...);
}
```

Not a big deal, just so it matches to formatting in the rest of the project (same with the one below)

Reply:
Yes, no problem, I'll fix that! Have you tested with the extension ? If you don't like the music that I picked, you can find thousands of songs in this site: https://modarchive.org/

Reply:
Perhaps, I can add sound effects to the game itself, like score and crash sounds ! (:

Reply:
I managed to get the modplayer extension compiled and installed but nothing would play. I've got a pretty custom OS build though so I've just assumed it was due to that.

And feel free to customise, this game was a bit of fun so I'm happy for you to go to town with it!

Reply:
Yes! Just to let you know, I'll speak in a conference in some weeks about this extension that I've created and I want to use this game as one of the examples!